### PR TITLE
ganeshShetty98/graduation

### PR DESCRIPTION
**Pull request by ganeshShetty98**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I have completed the form: `https://airtable.com/shrmuHxu38ZariKJi`
- [x] My application belongs to:
  - [x] **Tier 1** 📖: I added myself to the Year Book.
  - [x] **Tier 2** ✏️: I wrote a post on DEV about a project I built.
  - [x] **Tier 3** ✨: I want my project to be highlighted on stream.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

File: _data/ganeshShetty98/ganeshShetty98.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Shetty Ganeshprasad
+institution: SDM College of Engineering and Technology 
+profile_pic: ganesh.png 
+quote: Thanks to all teachers and my dear friends who encouraged and motivated me to get here. I'm going to miss you all!!!
```
Comment (mitulsavani) [inline _data/ganeshShetty98/ganeshShetty98.md:5]: Character count exceeds the limit; 100. You might want to modify this

**Comment by saarques** [comment]
Hey @ganeshShetty98 You might want to reduce size of your quote as Mitul said.

Also, please keep only one image as modified files should only be 2. Remove the extra image file.

Keep the name of the image same as provided in the markdown file.

**Comment by saarques** [comment]
> Also, please keep only one image as modified files should only be 2. Remove the extra image file.


Please consider this one too! :)

File: _data/ganeshShetty98/ganeshShetty98.md (hunk 1 above)
Comment (ganeshShetty98) [reply _data/ganeshShetty98/ganeshShetty98.md]: Yes  i have changed as u suggested and have replaced my image with new current one

Comment (ganeshShetty98) [reply _data/ganeshShetty98/ganeshShetty98.md]: okay thanks, I corrected it thanks.

Comment (ganeshShetty98) [reply _data/ganeshShetty98/ganeshShetty98.md]: if everything correct and reviewed why can't I merge.

Comment (mitulsavani) [reply _data/ganeshShetty98/ganeshShetty98.md]: You are fine don't worry about it, the merging team will handle that part

Comment (ganeshShetty98) [reply _data/ganeshShetty98/ganeshShetty98.md]: Okay Thanks for info.

**Pull request merged**